tests/test_aggregators.py

```diff
@@ -1,9 +1,37 @@
 import pytest
+from steering_vectors.aggregators import (
+    linear_aggregator,
+    logistic_aggregator,
+    mean_aggregator,
+    pca_aggregator,
+)
 import torch
-from steering_vectors.aggregators import mean_aggregator, pca_aggregator
 from torch.nn.functional import cosine_similarity
 
 
+def test_linear_aggregator() -> None:
+    delta = torch.randn(100)
+    pos = torch.randn(64, 100)
+    neg = pos - delta  # no noise because linear regression can't handle it
```

Comment:
I tried this as well, and it seems like the linear aggregator fails with any noise at all, even minute noise, which makes me worry something is wrong. I'm not sure if the `LinearRegression` model in scikit learn is suitable for classification problems like this. In the docs it shows a case where it's trying to fit a curve rather than do classification: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html. The  Logistic Regression docs, on the other hand, do talk about classification as the use case: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression

Reply:
Binary classification is indeed not the traditional use case for linear regression, but it should work, assuming the data is linearly separable. I'm also confused that adding even small deltas results in failure to recover the delta. After all logistic regression is just linear regression + a non-linearity and when we get vec, we drop the non-linearity. Anyway, if you're concerned that something is wrong, we can drop the linear regression aggregator since logistic regression is the standard approach.

Reply:
I'd support removing Linear Regression for now since it seems like something must not be working correctly. Probably most users would want logistic regression anyway, and it's still possible to implement for users by manually passing in their own custom `Aggregator` function.